### PR TITLE
Version number in footer

### PR DESCRIPTION
**Pull request by onursagir**
Before merging: 

- [x] href on `Laatste live-versie v1.0.0` needs to be updated @Henneddd / @sgort where should it point to.
- [x] If any would be nice to merge other footer related (small) issues. If @Henneddd can you name them?

### CODE REVIEW

File: apps/web/src/stories/footer.stories.tsx [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
 import { Footer } from '@/components/footer';
 import { Meta, StoryObj } from '@storybook/react';
+import { ApplicationMetadataContext } from '@/components/application-metadata-provider';
 
 const meta: Meta<typeof Footer> = {
   title: 'Footer',
   component: Footer,
+  decorators: [
+    (Story) => (
+      <ApplicationMetadataContext.Provider value={{ data: { version: '1.0.0', versionUpdatedAt: '2023-01-01' } }}>
```
Comment (sgort) [inline apps/web/src/stories/footer.stories.tsx:11]: Just wondering, hard coded version & date? I expect the version to match the release number?

**Review by sgort — APPROVED** [review]

File: apps/web/src/stories/footer.stories.tsx (hunk 1 above)
Comment (sgort) [reply apps/web/src/stories/footer.stories.tsx:11]: Guess this is related to `Laatste live-versie v1.0.0` task above?

Comment (sgort) [reply apps/web/src/stories/footer.stories.tsx:11]: So basically, the expected version number in the footer is the latest semantic versioning. This PR will update the current 1.8.0 (2023-03-03) release, isn't it? So the 1.8.1 or 1.9.0 wil be the one expected to be shown in the footer.

Comment (onursagir) [reply apps/web/src/stories/footer.stories.tsx:11]: No this is nit related to the task. The version number here is regarding the story, stories will usually have mocked values. The actual component will indeed have the actual version.

**Review by Henneddd — CHANGES_REQUESTED** [review]
- [x] I think the HREF should lead to our GitHub. The expected behavior would be to show our evolving code. 

- [x]  Small issue: icon alignment with text.
In any viewport, the icon is not aligned with the text, but it is especially noticeable in the mobile viewport.

**Review by Henneddd — APPROVED** [review]

**Pull request merged**